pkgs/development/python-modules/beaker/default.nix

```diff
@@ -62,10 +64,13 @@ buildPythonPackage rec {
   # Also disable a test in test_cache.py called "test_upgrade" because
   # it currently fails on darwin.
   # Please see issue https://github.com/bbangert/beaker/issues/166
+  # Also disable test failing with locale.Error: unsupported locale setting
   checkPhase = ''
+    export LOCALE_ARCHIVE="${glibcLocales}/lib/locale/locale-archive";
```

Comment:
Hmm, it seems like adding `glibcLocales` to `checkInputs` is a bit more customary in the code base:

```
$ git grep checkInputs | grep glibcLocale | wc -l
66
$ git grep LOCALE_ARCHIVE | wc -l              
45
```

Also, the "This PR does not cleanly list package outputs after merging" test failure is likely coming from the `${glibcLocales}` interpolation, as the error message in the logs says ["cannot coerce null to a string"](https://gist.github.com/GrahamcOfBorg/404709bde9555f52dc25cfac78bf3ace) and `glibcLocales` is defined as `null` when the host platform is not on glibc.

https://github.com/NixOS/nixpkgs/blob/27f084d817d6486ab0c2c23aff4305404646c2fa/pkgs/top-level/all-packages.nix#L12655

(sorry, didn't check this in my first review)

Reply:
@ento I've added `glibcLocales` into `checkInputs` as you suggested, thanks!